apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevDataMapperFerdigstillingVedtak.kt

```diff
@@ -322,6 +323,7 @@ class BrevDataMapperFerdigstillingVedtak(
             avdoede,
             datoVedtakOmgjoering,
             erMigrertYrkesskade.await(),
+            erYrkesskade = vilkaarsvurdering.await().isYrkesskade(),
```

Comment:
Har du sjekket med fag om det faktisk er vilkårsvurdering som er fasiten for yrkesskade? Ser at det er avhuking for dette i trygdetid også